.github/workflows/ci.yml

```diff
@@ -30,20 +30,14 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        rust: [stable]
 
-    name: Check feature permutations using Rust ${{ matrix.rust }} on ${{ matrix.os }}
+    name: Check feature permutations using Rust stable on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
 
-    env:
-      RUSTFLAGS: -Dwarnings
```

Comment:
While it sure is a pain that we have failing builds on every Rust update, that is quite intentional and we do want to deny warnings.

Reply:
> setting RUSTFLAGS: -Dwarnings is redundant, as all make jobs that are used already set it there

Reply:
We do have a few steps that are not using the makefile targets

Reply:
Iiiii don't belive so? I removed it from workflows that run `make checkall, `make checkfast` and `make testfast`. All of which already set `-Dwarnings`. (`make testfail` doesnt but in this case `checkfast` would fail first)